docs/CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Changelog
 
+## v5.1.0
+
+### Changed
+
+* ngclient default user-agent was updated from "tuf/x.y.z" to "python-tuf/x.y.z" (#2632)
+* max_root_rotations default value was bumped to 256 to prevent a too small value 
+  from creating issues in actual deployments were the embedded root is not easily
+  updateable (#2675)
+* Various infrastructure and documentation improvements
+
```

Comment:
Shall we mention the changes in the repository?
- https://github.com/theupdateframework/python-tuf/pull/2650

Reply:
this is s good call -- I think I looked at the git log with `git log --merges` but this likely won't show PRs merged with GitHubs _"squash and merge"_ ... I'll have to do another pass.

Reply:
fixed in the force-push